sfe/pages/unpause-form.html

```diff
@@ -1,64 +1,69 @@
-<!doctype html>
-<html dir="ltr" lang="en-US">
-    <header>
-        <title>Unpause - Self-Service Frontend</title>
-        {{ template "meta" }}
-    </header>
-    <body>
-    <div>
-    <h1>Action Required to Unpause Your ACME Account</h1>
+{{ template "header" }}
 
+<div class="section">
+    <h1>Action required to unpause your account</h1>
     <p>
-        You have been directed to this page because your Account ID {{ .AccountID }} 
-        is temporarily restricted from requesting new certificates for certain
-        identifiers including, but potentially not limited to, the following:
-        <ul>
-            {{ range $identifier := .Identifiers }}<li>{{ $identifier }}</li>{{ end }}
-        </ul>
+        You have been directed to this page because your ACME account (ID: {{
+        .AccountID }}) is temporarily restricted from requesting new
+        certificates for certain identifiers including, but not limited to, the
+        following:
     </p>
-    
-    <h2>Why Did This Happen?</h2> 
+    <ul>
+        {{ range $identifier := .Identifiers }}<li>{{ $identifier}}</li>{{ end }}
+    </ul>
+</div>
+
+<div class="section">
+    <h1>Why did this happen?</h1>
     <p>
         This often happens when domain names expire, point to new hosts, or if
         there are issues with the DNS configuration or web server settings.
-        These problems prevent your ACME client from successfully <a
-        href="https://letsencrypt.org/how-it-works/">validating control over the
-        domain</a>, which is necessary for issuing TLS certificates.
+        These problems prevent your ACME client from successfully
+        <a href="https://letsencrypt.org/how-it-works/">validating control over
+        the domain</a>, which is necessary for issuing TLS certificates.
     </p>
-    
-    <h2>What Can You Do?</h2>
+</div>
+
+<div class="faq-section section">
```

Comment:
Why "faq-section"? The other sections on this page are also questions, and they all seem likely to be frequently-asked. Maybe
```suggestion
<div class="section highlight">
```

Reply:
It's a remnant of how I was previously styling each Question and Answer section, but the final implementation is definitely a highlight. Thanks for pointing that out.